sdk/keyvault/azure-security-keyvault-perf/src/main/java/com/azure/security/keyvault/secrets/perf/core/SecretsTest.java

```diff
@@ -41,4 +45,13 @@ public SecretsTest(TOptions options) {
         secretClient = builder.buildClient();
         secretAsyncClient = builder.buildAsyncClient();
     }
+
+    protected Mono<Void> deleteAndPurgeSecretsAsync(String ... names) {
+        return Flux
+            .fromArray(names)
+            .flatMap(name -> secretAsyncClient
```

Comment:
The goal here is to ignore exceptions thrown if the secret being deleted does not exist.  Does this reactor code look correct?  It seems to work in a basic test.

Reply:
I think it should work. There might be some ways to make this a bit more "reactive" but it should do the work. Any comments @srnagar @alzimmermsft?

Reply:
This looks good to me. The only thing to verify is the semantics of delete and purge methods when the resource is not found. Does it throw `ResourceNotFoundException` or some other type of exception (HttpResponseException?) or return a 200? Also, log the name of the secret that was not found.

```
.beginDeleteSecret(name)
.doOnError(name -> logger.info(name + " not found")
.onErrorResume(ResourceNotFoundException.class, e -> Mono.empty())
```

Reply:
I don't think `PerfStressTest` has a `logger` object to use, should I just log to `System.out`?